docs/source/bricks.rst

```diff
@@ -77,6 +77,22 @@ Examples:
       text = f.read()
   elements = partition_email(text=text)
 
+``extract_attachment_info``
+----------------------
+
+The ``extract_attachment_info`` function takes an ``email.message.Message`` object
+as input and returns the a list of dictionaries containing the attachment information,
+such as ``filename``, ``size``, ``payload``, etc.
```

Comment:
Probably self-explanatory from  the name `output_dir`, but I'd also mention that the attachments get saved to `output_dir`.